changelog/unreleased/issue-2491

```diff
@@ -0,0 +1,9 @@
+Bugfix: Don't require `self-update --output` placeholder file
+
+`restic self-update --output /path/to/new-restic` used to require that
+_new-restic_ was an existing file, to be overwritten. Now it's
```

Comment:
Please remove the _ around new-restic as the changelog isn't Markdown. It's perfectly fine to put commands in some kind of delimiter though, so we don't need to change that.

Reply:
Done.